### DISCUSSION TITLE
Rancher Desktop 1.2.1 hangs

### DISCUSSION THREAD

**Discussion by cdprete** [rancher-sandbox/rancher-desktop#1985] [Q&A]
Hi.
I've just installed Rancher Desktop on a Windows 10 laptop, but this hangs in "Waiting for nodes".
All the logs file are empty even when the debug is enabled and the Images tab stays in "Waiting for image manager to be ready".
What can I do?

The machine has an i5-7200U CPU with 8GB of RAM.

**Comment by adamkpickering** [comment]
Maybe you are maxing out your memory usage? 8GB should be enough, but if you're running other applications it can get eaten up fast. I've definitely used up all of my memory while running RD, and I have 16 GB of it on my machine.

Also, have you tried reinstalling? Have you tried a factory reset (you can do this via the "Troubleshooting" tab)?

**Reply by cdprete** [reply]
Hi.
I've tried re-installing it as well as with the factory reset.  No luck.
Reinstalling the entire OS from scratch made it work, though. Even if now I don't have any other application installed in order to be on the safe side.

On another machine (still 8GB but an i7 CPU) the UI doesn't even show off. The icon flashes for a moment in the try bar an that's all.

**Reply by adamkpickering** [reply]
Are any logs produced? If so, can you post them here? If you are able to enable debug mode before grabbing logs, that would help too.

**Reply by cdprete** [reply]
Hi.
No logs were produced unfortunately. On the i7 machine I managed to make it run by installing all the WIndows updates, re-installing the WSL2 update (although it was already there), and by playing around with enabling&disabling HyperV&Co. from the Windows features menu.
